Realm.podspec

```diff
@@ -82,6 +83,7 @@ Pod::Spec.new do |s|
                               'include/**/RLMObjectStore.h',
                               'include/**/RLMObject_Private.h',
                               'include/**/RLMOptionalBase.h',
+                              'include/**/RLMPushClient_Private.hpp',
```

Comment:
This should not be here. This section is things in the Realm.Private module, which is headers exposed to RealmSwift and nothing else. RealmSwift doesn't need this file, and if it did it'd need to be added in other places too.

Reply:
Removed